bridgetown-routes/lib/bridgetown-routes/manifest.rb

```diff
@@ -32,7 +32,19 @@ def generate_manifest(site) # rubocop:todo Metrics/AbcSize, Metrics/CyclomaticCo
                 ":#{Regexp.last_match(1)}"
               end
 
-              [file, file_slug, segment_keys]
+              # generate localized file slugs
+              localized_file_slugs = []
+              site.config.available_locales.each do |locale|
```

Comment:
We could switch this to using map for simpler code, aka `localized_file_slugs = site.config.available_locales.map do |locale|`

Reply:
Thanks! I'm glad you liked the implementation. This change looks cleaner to me. I'll fix it right away!